inst/examples/dashboard/app.R

```diff
@@ -5,6 +5,9 @@ library(shiny.fluent)
 library(shiny.router)
 library(shiny)
 library(stringi)
+library(bslib)
+library(imola)
```

Comment:
When you see a sorted list, please keep it so. Don't just add entries to the end; add them so the library calls stay in alphabetical order. This makes it easier to see at a glance if a given library is included and can make resolving merge conflicts easier too.